integration/localnet/bootstrap.go

```diff
@@ -153,21 +154,24 @@ func main() {
 
 	fmt.Println("Node bootstrapping data generated...")
 
-	var secureAccessNodeID string
+	// gather access node IDS for LN/SN nodes
+	var accessNodeIDS strings.Builder
 	for i, c := range containers {
 		fmt.Printf("%d: %s", i+1, c.Identity().String())
 		if c.Unstaked {
 			fmt.Printf(" (unstaked)")
 		}
 
 		if c.Role == flow.RoleAccess && !c.Unstaked {
-			secureAccessNodeID = c.NodeID.String()
+			accessNodeIDS.WriteString(fmt.Sprintf("%s,", c.NodeID.String()))
 		}
 
 		fmt.Println()
 	}
 
-	services := prepareServices(containers, secureAccessNodeID)
+	// remove extra comma at the end of string
+	anIDS := accessNodeIDS.String()[:accessNodeIDS.Len()-1]
```

Comment:
Can use `Join` to make this a bit cleaner
```go
var accessNodeIDs []string
for ... {
   // ...
   accessNodeIDS = append(accessNodeIDS, c.NodeID.String())
}
anIDs := strings.Join(accessNodeIDS, ",")
```

Reply:
https://github.com/onflow/flow-go/pull/1395/commits/63038258db5767cf410d7b0178d2c13bf2cf1e35